lib/server/auth.js

```diff
@@ -114,48 +115,75 @@ class DataAccess extends EventEmitter {
     });
     LOGGER.debug('ACCESS GROUP IDS: ' + agroupIds.join(' '));
 
-    let authurl = config.provide().get('authurl');
+    let auth_server = config.provide().get('auth_server');
+    let authurl = auth_server.url;
     let reqbody = {groups: agroupIds};
-    let reqopts = url.parse(authurl);
-    reqopts.method = 'POST';
+    let reqopts = {};
+    reqopts.url     = url.parse(authurl);
+    reqopts.method  = 'POST';
+    reqopts.headers = {
+      'Content-Type': 'application/json'
+    };
+
+    if ( authurl.startsWith('https://') ) {
```

Comment:
Is this check needed? Process should exit at lib/config_helpers.js#_validateAuthConfiguration if protocol isn't https... If we do want to check again, we should probably throw an error if execution has somehow reached here without already throwing